code/modules/jobs/job_types/security.dm

```diff
@@ -124,7 +124,7 @@ Detective
 
 	outfit = /datum/outfit/job/detective
 
-	access = list(access_sec_doors, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
+	access = list(access_sec_doors, access_security, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
```

Comment:
in essence this encourages powergaming, because they aren't being given the tools they need, so they are gonna find a way to get them that will land them in trouble with server staff